rasa/core/exporter.py

```diff
@@ -241,6 +241,11 @@ async def _fetch_events_within_time_range(self) -> AsyncIterator[Dict[Text, Any]
                 continue
 
             events = self._get_events_for_conversation_id(_events, conversation_id)
+
+            # the order of events was changed after ATO-2192
```

Comment:
We should link to the GH PR in the comment.

Reply:
Done